fvm/src/machine/engine.rs

```diff
@@ -118,6 +118,8 @@ pub fn default_wasmtime_config() -> wasmtime::Config {
     // wasmtime default: 512KiB
     // Set to something much higher than the instrumented limiter.
     // Note: This is in bytes, while the instrumented limit is in stack elements
+    #[cfg(feature = "testing")]
+    c.async_stack_size(4 << 20).unwrap();
```

Comment:
What's going on here?

Reply:
When the integration test code is running, the `async` feature is on in wasmtime and it checks if the `async_stack_size` is at least as big as the stack size that is set in the following line, if it is smaller it panics.

Reply:
This issue has been fixed on master, you shouldn't need to change this.